YIF_XUnitTests/Unit/YIF.Core.Service/Concrete/Services/SuperAdminServiceTests.cs

```diff
@@ -285,7 +285,7 @@ public async Task DisableAdmin_ReturnsSuccessEnableMessage()
         {
```

Comment:
please rename variable "a" to "result" at lines 277, 280, 300, 303. Because all variables should have meaningful names.

Reply:
done